src/index.html

```diff
@@ -66,7 +80,7 @@
     <div class="clearfix">
       <div class="pull-left text-muted" data-ng-controller="LanguageCtrl">
         <div uib-dropdown class="dropup">
-          <span>Language: </span>
+          <span>{{"LANGUAGE"|translate}}: </span>
```

Comment:
In other places you are using the `translate` directive instead of the `translate` filter. Let's do the same here?